Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:22.04
+
+WORKDIR /metachain 
+
+# Requires copy the binary to `build` folder beforehand
+COPY build/* /metachain 
```

Comment:
```suggestion
COPY target/release/meta-node /metachain 
```

I'm wondering if we should do this instead, so we can omit `- name: Save binary`, since we're not saving anything but rather moving it?

Reply:
the `target` storage is not quite friendly :sweat_smile: 
![image](https://user-images.githubusercontent.com/8097901/186442882-d32cb27f-ec67-4ba2-ba88-b646e159bf7e.png)

Reply:
Cries LOL